playback/src/main/kotlin/voice/playback/session/PlaybackService.kt

```diff
@@ -57,14 +57,13 @@ class PlaybackService : MediaLibraryService() {
     release()
   }
 
-  override fun onGetSession(controllerInfo: MediaSession.ControllerInfo): MediaLibrarySession? {
-    return session.takeUnless { session ->
-      session.invokeIsReleased
-    }.also {
-      if (it == null) {
-        Logger.e("onGetSession returns null because the session is already released")
-      }
-    }
+  override fun onGetSession(controllerInfo: MediaSession.ControllerInfo): MediaLibrarySession {
+    if (session.invokeIsReleased)
+      rootComponentAs<PlaybackComponent.Provider>()
```

Comment:
And are you sure this is necessary? I'm hesitant to merge this because this will literally create new instances of the whole player stuff - while not releasing the old things.
I fear that this might lead to several players playing at the same time under some circumstances

Reply:
A new session is only created in a situation when the service is not destroyed after a `release` call. I face this situation when I run Voice from SimpleWear and then close it. Now in such a situation playback cannot be started even by the play button in the player activity (or the play button in the books list). I think this behaviour is not more desirable.
By the way, in the comment in `onTaskRemoved` you have written that releasing the session is required to prevent showing a dead notification. Later, in a comment to my previous PR, you've mentioned having fixed the notification being unresponsive. So, is the dead notification still an issue, and is the `release` call in `onTaskRemoved` still required? It seems to me that only calling `release` from `onDestroy` would obviate the sole need to check if the session state.

Reply:
And which conditions do you mean? Is that `onGetSession` being called from another thread than `onCreate` that does not see the session created there?

Reply:
Also, I don't quite understand what you mean by 'not releasing the old stuff' - a new session is only created if the old one has already been released.

Reply:
Any updates?:)